main.go

```diff
@@ -43,28 +44,31 @@ func init() {
 		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
 	flag.BoolVar(&debug, "debug", false, "Debug logging")
 	flag.Parse()
-
-	_ = addonmgrv1alpha1.AddToScheme(scheme)
-	// +kubebuilder:scaffold:scheme
```

Comment:
This is required by kubebuilder

Reply:
We already customized the scheme. do not really need previous  one any more